test/qunit/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <script src="../../dist/js-reporters.js"></script>
+    <link href="../../node_modules/qunitjs/qunit/qunit.css" rel="stylesheet" type="text/css"/>
+    <script src="../../node_modules/qunitjs/qunit/qunit.js"></script>
+    <script src="tests.js"></script>
+
+    <script>
+        var adapter = new JsReporters.QUnitAdapter();
+
+
+        var tapReporter = new JsReporters.TapReporter(adapter);
```

Comment:
This looks odd. If the resulting object isn't used at all, we shouldn't be calling a constructor. Might as well call `JsReporters.tapReporter()` or `JsReporters.TapReporter.init()`.